tests/boot/boot_from_pxe.pm

```diff
@@ -21,27 +21,58 @@ use testapi;
 use bootloader_setup qw(bootmenu_default_params specific_bootmenu_params);
 use registration 'registration_bootloader_cmdline';
 use utils 'type_string_slow';
+use backend::ipmi 'ipmitool';
+
+# Variables are present in vars.json
+# IPMI_HOSTNAME, IPMI_USER, IPMI_PASSWORD
+my $ipmi_host = {
+    hostname => get_required_var('IPMI_HOSTNAME'),
+    username => get_required_var('IPMI_USER'),
+    userpass => get_required_var('IPMI_PASSWORD'),
+};
+
+sub reactivate_sol {
+    diag("IPMI host name is $ipmi_host->{hostname}");
+    diag("IPMI Activate SOL");
+    my $rc = system("ipmitool -I lanplus -H $ipmi_host->{hostname} -U $ipmi_host->{username} -P $ipmi_host->{userpass} -vv sol activate");
```

Comment:
I guess we could extract common part of the command to make it more readable.

Reply:
Yup, nice catch! thanks!